image/tarexport/save.go

```diff
@@ -191,15 +189,13 @@ func (s *saveSession) save(outStream io.Writer) error {
 	manifestFileName := filepath.Join(tempDir, manifestFileName)
 	f, err := os.OpenFile(manifestFileName, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0644)
 	if err != nil {
-		f.Close()
 		return err
 	}
+	defer f.Close()
```

Comment:
this is for more consistent output of `docker save`. You can mark the time of the tar archives based on the createdAt in its manifest. But for files like `repositories` could vary base on images being output, so nil it to epoch.
With the content addressibility features, it could enable `docker save` outputs to have same checksums for same content. previously it was always changing.